File: cakk-domain/mysql/src/main/java/com/cakk/domain/mysql/dto/param/shop/UpdateShopAddressParam.java
package com.cakk.domain.mysql.dto.param.shop;

import org.locationtech.jts.geom.Point;

import lombok.Builder;

import com.cakk.domain.mysql.entity.user.User;

@Builder
public record UpdateShopAddressParam(
	String shopAddress,
	Point location,
	User user,
	Long cakeShopId
) {
}

File: cakk-domain/mysql/src/main/java/com/cakk/domain/mysql/dto/param/shop/UpdateShopAddressParam.kt
package com.cakk.domain.mysql.dto.param.shop

import com.cakk.domain.mysql.entity.user.User
import org.locationtech.jts.geom.Point

data class UpdateShopAddressParam(
	val shopAddress: String,
	val location: Point,
	val user: User,
	val cakeShopId: Long
)
